src/subcommand.rs

```diff
@@ -92,6 +99,7 @@ impl Subcommand {
                 .unwrap()
                 .join("target")
         });
+        let target_dir = std::fs::canonicalize(target_dir).unwrap();
```

Comment:
I prefer to use `dunce` here because UNC has bitten us many times in `android-ndk-rs`. :warning: this code hasn't been tested on Windows yet!

@dvc94ch Are you okay with me adding that crate and replacing the only other `canonicalize` call in `utils::find_package` with it?